docs/gl_objects/project_secure_files.rst

```diff
@@ -0,0 +1,47 @@
+####################
+Project Secure Files
+####################
+
+Project secure files
+====================
+
+References
+----------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.ProjectSecureFile`
+  + :class:`gitlab.v4.objects.ProjectSecureFileManager`
+  + :attr:`gitlab.Gitlab.project_secure_files`
+
+* GitLab API: https://docs.gitlab.com/ee/api/secure_files.html
+
+Examples
+--------
+
+Get a project secure file::
+
+    secure_files = gl.projects.get(1, lazy=True).secure_files.get(1)
+    print(secure_files.name)
+
+List project secure files::
+
+    secure_files = gl.projects.get(1, lazy=True).secure_files.list()
+    print(secure_files[0].name)
+
+Create project secure file::
+
+    secure_file = gl.projects.get(1).secure_files.create({"name": "test", "file": "secure.txt"})
+
+Download a project secure file::
+
+    content = secure_file.download()
+    print(content)
```

Comment:
Shouldn't you use logging instead of the print command?

Reply:
This is consistent with the rest of the documentation, and only serves as a reference. Documenting the use of appropriate logging mechanisms seems out of scope.

Reply:
Apologies for my oversight. I was under the impression that logging is already being used in the project and we want to avoid using the `print` command.  Replacing the `print` commands with logging is probably not in the scope of this PR.